### PR TITLE
SM-1159: Rename Service Accounts to Machine Accounts

### PR DESCRIPTION
**Pull request by coltonhurst**
## Type of change

- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Rename Service Accounts to Machine Accounts. In this PR we create new i18n entries, preserving the Service Account ones purposely for easy rollback and to keep the translations. Please note, this refactor only applies to text that is visible to the user, for now the code base still refers to these types of accounts as Service Accounts.

Server PR: https://github.com/bitwarden/server/pull/3958

## Code changes

Only added new i18n entries and swapped the old ones for the new ones. Everything should be the same except Service Account becomes Machine Account, with the casing kept based on the initial message.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/app/billing/organizations/sm-adjust-subscription.component.html [hunk 1]
```diff
@@ -44,8 +44,8 @@
     />
     <bit-hint>
       <div>
```
Comment (cyprain-okeke) [inline apps/web/src/app/billing/organizations/sm-adjust-subscription.component.html:46]: Should we not be remaining the i18n:options.baseServiceAccountCount  for uniformity

File: apps/web/src/app/billing/shared/sm-subscribe.component.html [hunk 2]
```diff
@@ -20,10 +20,10 @@ <h4 bitTypography="h4">{{ "secretsManagerForPlan" | i18n: planName }}</h4>
         <ng-template #unlimitedProjects>
           <li>{{ "unlimitedProjects" | i18n }}</li>
         </ng-template>
-        <li>{{ "serviceAccountsIncluded" | i18n: serviceAccountsIncluded }}</li>
+        <li>{{ "machineAccountsIncluded" | i18n: machineAccountsIncluded }}</li>
         <li *ngIf="product != productTypes.Free">
           {{
-            "additionalServiceAccountCost" | i18n: (monthlyCostPerServiceAccount | currency: "$")
+            "additionalMachineAccountCost" | i18n: (monthlyCostPerServiceAccount | currency: "$")
```
Comment (cyprain-okeke) [inline apps/web/src/app/billing/shared/sm-subscribe.component.html:26]: same here too

File: apps/web/src/app/billing/organizations/sm-adjust-subscription.component.html (hunk 1 above)
Comment (coltonhurst) [reply apps/web/src/app/billing/organizations/sm-adjust-subscription.component.html:46]: Hey @cyprain-okeke thanks for pointing this out. `options` is just a field under `SecretsManagerAdjustSubscriptionComponent` in `apps/web/src/app/billing/organizations/sm-adjust-subscription.component.ts`.

This PR only reflects changes to text that users will see, we are not removing the service account types or names in the rest of the code at this point.

File: apps/web/src/app/billing/shared/sm-subscribe.component.html (hunk 2 above)
Comment (coltonhurst) [reply apps/web/src/app/billing/shared/sm-subscribe.component.html:26]: Hey @cyprain-okeke thanks for pointing this out. `monthlyCostPerServiceAccount` is just a getter function under `SecretsManagerSubscribeComponent` in `apps/web/src/app/billing/shared/sm-subscribe.component.ts`.

This PR only reflects changes to text that users will see, we are not removing the service account types or names in the rest of the code at this point.

**Review by r-tome — COMMENTED** [review]
Looks good to me but shouldn't we delete the old i18n entries if they're not used anymore?

**Review by cyprain-okeke — COMMENTED** [review]
Looks goos for billing

**Comment by coltonhurst** [comment]
> Looks good to me but shouldn't we delete the old i18n entries if they're not used anymore?

@r-tome for now we want to keep the old entries. The main reason for this is if we want to switch back to service accounts, this will be easy, especially as it means we can also keep the translations.

**Review by r-tome — APPROVED** [review]
LGTM!

**Comment by coltonhurst** [comment]
Merged branch `main` into this branch `sm/sm-1159`, there were conflicts (now resolved) in the `apps/web/src/locales/en/messages.json` file.

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Review by dani-garcia — APPROVED** [review]

**Review by cturnbull-bitwarden — DISMISSED** [review]
Billing changes look good!

**Review by r-tome — APPROVED** [review]

**Review by Thomas-Avery — APPROVED** [review]

**Review by dani-garcia — APPROVED** [review]

**Review by cturnbull-bitwarden — APPROVED** [review]

**Review by vincentsalucci — APPROVED** [review]
Reviewed AC owned file.

**Comment by coltonhurst** [comment]
Note: QA will test on `main`.

**Pull request merged**